Formula/podman.rb

```diff
@@ -30,5 +33,9 @@ def install
   test do
     assert_match "podman version #{version}", shell_output("#{bin}/podman -v")
     assert_match(/Error: Cannot connect to the Podman socket/i, shell_output("#{bin}/podman info 2>&1", 125))
+    if Hardware::CPU.intel?
+      machineinit_output = shell_output("podman machine init --image-path fake-testimage123 fake-testvm123 2>&1", 125)
+      assert_match "Error: open fake-testimage123: no such file or directory", machineinit_output
```

Comment:
Does this return a different error if `podman` can't find `qemu` or isn't able to use `qemu` properly?

Reply:
yep. it'll return `Error: exec: "qemu-system-x86_64": executable file not found in $PATH` if qemu isnt installed